x-pack/plugins/cloud_security_posture/public/pages/benchmarks/benchmarks_table.tsx

```diff
@@ -67,6 +78,39 @@ const IntegrationButtonLink = ({
   );
 };
 
+const ErrorMessageComponent = (error: ErrorProps) => (
+  <FullSizeCenteredPage>
+    <EuiEmptyPrompt
+      color="danger"
+      iconType="warning"
+      data-test-subj={ERROR_STATE_TEST_SUBJECT}
+      title={
+        <h2>
+          <FormattedMessage
+            id="xpack.csp.benchmarks.benchmarksTable.errorRenderer.errorTitle"
+            defaultMessage="We couldn't fetch your cloud security posture benchmark data"
+          />
+        </h2>
+      }
+      body={
+        isCommonError(error) ? (
+          <p>
+            <FormattedMessage
+              id="xpack.csp.benchmarks.benchmarksTable.errorRenderer.errorDescription"
+              defaultMessage="{error} {statusCode}: {body}"
```

Comment:
It seems there's nothing to translate here

Reply:
don't we need to translate the body if it exist ?